modules/create-lokistack-cr-cli.adoc

```diff
@@ -0,0 +1,85 @@
+
+.Prerequisites
+
+* You have administrator permissions.
+* You have installed the {loki-op}.
+* You have installed the {oc-first}.
+
+.Procedure
+
+. Create a `LokiStack` CR:
++
+.Example `LokiStack` CR
+[source,yaml]
+----
+apiVersion: loki.grafana.com/v1
+kind: LokiStack
+metadata:
+  name: logging-loki
+  namespace: openshift-logging
+spec:
+  size: 1x.small # <1>
+  storage:
+    schemas:
+    - version: v12
+      effectiveDate: "2022-06-01"
+    secret:
+      name: logging-loki-s3 # <2>
+      type: s3 # <3>
+  storageClassName: <storage_class_name> # <4>
+  tenants:
+    mode: openshift-logging
+----
+<1> Supported size options for production instances of Loki are `1x.small` and `1x.medium`.
+<2> Enter the name of your log store secret.
+<3> Enter the type of your log store secret.
+<4> Enter the name of an existing storage class for temporary storage. For best performance, specify a storage class that allocates block storage. Available storage classes for your cluster can be listed using `oc get storageclasses`.
+
+. Apply the `LokiStack` CR:
++
+[source,terminal]
+----
+$ oc apply -f <filename>.yaml
+----
+
+.Verification
+
+* Verify the installation by listing the pods in the `openshift-logging` project by running the following command and observing the output:
++
+[source,terminal]
+----
+$ oc get pods -n openshift-logging
+----
++
+You should see several pods for components of the {logging}, similar to the following list:
++
+.Example output
+[source,terminal]
+----
+NAME                                               READY   STATUS    RESTARTS   AGE
+cluster-logging-operator-fb7f7cf69-8jsbq           1/1     Running   0          98m
+collector-222js                                    2/2     Running   0          18m
+collector-g9ddv                                    2/2     Running   0          18m
+collector-hfqq8                                    2/2     Running   0          18m
+collector-sphwg                                    2/2     Running   0          18m
+collector-vv7zn                                    2/2     Running   0          18m
+collector-wk5zz                                    2/2     Running   0          18m
+logging-view-plugin-6f76fbb78f-n2n4n               1/1     Running   0          18m
+lokistack-sample-compactor-0                       1/1     Running   0          42m
+lokistack-sample-distributor-7d7688bcb9-dvcj8      1/1     Running   0          42m
+lokistack-sample-gateway-5f6c75f879-bl7k9          2/2     Running   0          42m
+lokistack-sample-gateway-5f6c75f879-xhq98          2/2     Running   0          42m
+lokistack-sample-index-gateway-0                   1/1     Running   0          42m
+lokistack-sample-ingester-0                        1/1     Running   0          42m
+lokistack-sample-querier-6b7b56bccc-2v9q4          1/1     Running   0          42m
+lokistack-sample-query-frontend-84fb57c578-gq2f7   1/1     Running   0          42m
```

Comment:
Just related to the point I mentioned above, since we have mentioned Lokistack name as  'logging-loki', we should have the same for the components as well and not lokistack-sample.
Suggest to change it to 
```
NAME                                           READY   STATUS    RESTARTS   AGE
cluster-logging-operator-78fddc697-mnl82       1/1     Running   0          8m46s
logging-loki-compactor-0                       1/1     Running   0          2m37s
logging-loki-distributor-b85b7d9fd-25j9g       1/1     Running   0          2m37s
logging-loki-distributor-b85b7d9fd-xwjs6       1/1     Running   0          2m37s
logging-loki-gateway-7bb86fd855-hjhl4          2/2     Running   0          2m37s
logging-loki-gateway-7bb86fd855-qjtlb          2/2     Running   0          2m37s
logging-loki-index-gateway-0                   1/1     Running   0          2m37s
logging-loki-index-gateway-1                   1/1     Running   0          2m4s
logging-loki-ingester-0                        1/1     Running   0          2m37s
logging-loki-ingester-1                        1/1     Running   0          81s
logging-loki-querier-f5cf9cb87-9fdjd           1/1     Running   0          2m37s
logging-loki-querier-f5cf9cb87-fp9v5           1/1     Running   0          2m37s
logging-loki-query-frontend-58c579fcb7-lfvbc   1/1     Running   0          2m37s
logging-loki-query-frontend-58c579fcb7-tjf9k   1/1     Running   0          2m37s
```